### DISCUSSION TITLE
Need Help in setting up in local host

### DISCUSSION THREAD

**Discussion by jot-s-bindra** [optuna/optuna#5002] [Q&A]
I was following the `contributing.md` file to set up Optuna in my local host but I received this error on my terminal:

```bash
./formats.sh
black succeeded.
blackdoc succeeded.
flake8 succeeded.
isort succeeded.
optuna/storages/_rdb/models.py:40: error: Name "_Column" already defined on line 37  [no-redef]
optuna/storages/_rdb/models.py:40: note: Error code "no-redef" not covered by "type: ignore" comment
Found 1 error in 1 file (checked 339 source files)
mypy failed.
```
After some research, I found out this is because of a conflicting name "_Column" on line 37. How to fix this because I don't think there will be an issue in setting up on local host; I might be messing something.

**Comment by nzw0301** [comment]
Could you share more context? Did you apply a change to `optuna/storages/_rdb/models.py`? In addition, What is the mypy version?

**Reply by jot-s-bindra** [reply]
I didnt change anything in the repo , just cloned it and followed contributing.md steps ,during formats.sh i got this error
